tests/test_login_path.py

```diff
@@ -90,3 +92,42 @@ def test_corrupted_pad():
     for word in ('[test]', 'password', 'host', 'port'):
         assert word in contents
     assert 'user' not in contents
+
+
+def test_get_mylogin_cnf_path():
+    """Tests that the path for .mylogin.cnf is detected."""
+    original_env = None
+    if 'MYSQL_TEST_LOGIN_FILE' in os.environ:
+        original_env = os.environ.pop('MYSQL_TEST_LOGIN_FILE')
+    is_windows = sys.platform == 'win32'
+
+    login_cnf_path = get_mylogin_cnf_path()
+
+    if original_env is not None:
+        os.environ['MYSQL_TEST_LOGIN_FILE'] = original_env
+
+    if login_cnf_path is not None:
+        assert login_cnf_path.endswith('.mylogin.cnf')
+
+        if is_windows is True:
+            assert 'MySQL' in login_cnf_path
+        else:
+            home_dir = os.path.expanduser('~')
+            assert login_cnf_path.startswith(home_dir)
+
+
+def test_alternate_get_mylogin_cnf_path():
+    """Tests that the alternate path for .mylogin.cnf is detected."""
+    original_env = None
+    if 'MYSQL_TEST_LOGIN_FILE' in os.environ:
+        original_env = os.environ.pop('MYSQL_TEST_LOGIN_FILE')
+
+    temp_fh, temp_path = tempfile.mkstemp()
```

Comment:
@tsroten I think Github don't notify users when there is a update in a comment. As you can see, yesterday I've updated my comment  to `_, temp_path = temp_file.mkstemp()` because is the more _pythonic_ way. If you don't mind, I can push this small change and sorry to bother you with these things.